### ISSUE TITLE
Not working with v2.0.1

### ISSUE THREAD

**Issue by fandiahm** [michielkempen/nova-order-field#8]
I guess this package is not working with v2.0.1. Just downgrade nova to v2.0.0 and it works.

**Comment by digitalunityca** [comment]
+1 here

**Comment by morphsteve** [comment]
Since 2.0.1 passing the resource name as a request param will no longer work, as Nova expects the resource name to be a route parameter. See `Laravel\Nova\Http\Requests\InteractsWithResources`:

```php
public function resource()
{
    return tap(Nova::resourceForKey($this->route('resource')), function ($resource) {
        abort_if(is_null($resource), 404);
        abort_if(! $resource::authorizedToViewAny($this), 403);
    });
}
```

**Comment by michielkempen** [comment]
This problem is fixed in version 2.0.0 of the package.
